src/main/java/jenkins/branch/UntrustedBranchProperty.java

```diff
@@ -120,6 +121,7 @@ public UntrustedBranchProperty(String[] publisherWhitelist) {
     /**
      * Our {@link Descriptor}.
      */
+    @Symbol("untrustedBranches")
```

Comment:
I should have thought about this more before I recommended `untrustedBranches`, but since the base type is `BranchProperty` I think the best symbol here is just `untrusted`.